honeybee_radiance_command/rmtxop.py

```diff
@@ -0,0 +1,408 @@
+"""rmtxop command."""
+from .options.rmtxop import RmtxopOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Rmtxop(Command):
+    """Rmtxop command.
+
+    Concatenate, add, multiply, divide, transpose, scale, and convert matrices. The
+    implementation here allows for matrix operations on upto three matrices at the same
+    time. While the Radiance binary allows for even more, this restriction has been
+    imposed here to keep the code somewhat clean.
+    """
+
+    __slots__ = ('_mtx1', '_mtx1_trnsps', '_mtx1_trnsfrm', '_mtx1_scl',
+                 '_mtx2', '_mtx2_trnsps', '_mtx2_trnsfrm', '_mtx2_scl',
+                 '_mtx3', '_mtx3_trnsps', '_mtx3_trnsfrm', '_mtx3_scl',
+                 '_mtx12_oprtr', '_mtx23_oprtr','_mtx_count')
+
+    def __init__(self, options=None, output=None, mtx1=None, mtx1_trnsfrm=None,
```

Comment:
Hi @built-tools! I read your comment here (https://github.com/ladybug-tools/honeybee-radiance-command/pull/218#discussion_r769027683). I'm still not convinced that this is the cleaner solution. I think you can still do a decent job in catching errors and reporting issues.

If we decide to keep it like this, then my suggestion is to add a documentation for each arg so one would know how to use them.

Reply:
@mostaphaRoudsari Let me try to get a list-based implementation going. If I cannot get it done before the weekend, then I will add documentation for this one (which is far easier to do :P ). We have everything else in place, so I am hoping to get at least some version of the multi-window group script done during the weekend.